modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -14,36 +14,39 @@ When an app attempts to consume the protected API resource, the rate limiting SL
 
 SLA-based rate limiting requires adding a RAML or OAS snippet to your API. This procedure demonstrates adding a RAML snippet.
```

Comment:
One overall comment. The uber steps: Add the Required RAML Snippet, Add the SLA Tier and Apply the Policy if they are in Chrono order, should we mention that like here: https://docs.mulesoft.com/api-manager/2.x/rest-validator-extensionhttps://docs.mulesoft.com/api-manager/2.x/rest-validator-extension

Reply:
Fixed!